dataplane/internal/engine/engine.go

```diff
@@ -80,7 +83,7 @@ func New(ctx context.Context, id string, c fwdpb.ForwardingClient) (*Engine, err
 			},
 		},
 	}
-	if _, err := c.TableCreate(ctx, v4FIB); err != nil {
+	if _, err := e.Engine.TableCreate(ctx, v4FIB); err != nil {
```

Comment:
Reading this is a bit confusing: an engine that has an engine. Can add a doc comment for this embedded variable how it's different.

Reply:
agreed, renamed forwarding.Engine to forwarding.Server